Formula/c/cspell.rb

```diff
@@ -0,0 +1,20 @@
+class Cspell < Formula
+  desc "Spell checker for code"
+  homepage "https://cspell.org"
+  url "https://registry.npmjs.org/cspell/-/cspell-8.17.4.tgz"
+  sha256 "706660433488a9889c2b81e1a10c86a0595b742ccd4815077af749f3aaa70d61"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *std_npm_args
+    bin.install_symlink Dir["#{libexec}/bin/cspell"]
```

Comment:
Just add a comment that we intentionally don't link `cspell-esm`, I think.
```suggestion
    # comment here
    bin.install_symlink libexec/"bin/cspell"
```
We also don't need `Dir` anymore.